tests/integration/mock/CodeActAgent/test_edits/prompt_003.log

```diff
@@ -347,12 +353,12 @@ Enjoy!""")
 ----------
 
 OBSERVATION:
-[File: /workspace/bad.txt (4 lines total after edit)]
-1|This is a stupid typo.
-2|Really?
-3|No more typos!
-4|Enjoy!
-[File updated. Please review the changes and make sure they are correct (correct indentation, no duplicate lines, etc). Edit the file again if necessary.]
+[File: /workspace/bad.txt (4 lines total after edit)]
+1|This is a stupid typo.
+2|Really?
+3|No more typos!
+4|Enjoy!
+[File updated. Please review the changes and make sure they are correct (correct indentation, no duplicate lines, etc). Edit the file again if necessary.]
```

Comment:
weird, I don't understand why GitHub is showing a diff here... I don't see any changes.

Reply:
Probably because I had to restart from scratch with a `main`-level branch and re-apply my changes. Might just be an artifact.

Reply:
Or maybe some whitespace. If I choose the setting "Hide whitespace" on GitHub, then reload, I no longer see these particular lines in the diff.

Reply:
Probably difference between `\n` and `\r\n`, coz it doesn't seem there's any trailing space.